fabfile/setup.py

```diff
@@ -39,13 +39,16 @@ def create_ansible_env():
     Creates the ansible environment
     :return: Void
     """
+    #TODO: `"mkvirtualenv"` and `"workon"` don't work in `local` call on some platforms
     with bash():
         with settings(warn_only=True):
             local('mkvirtualenv ansible')
     with ansible():
         with settings(warn_only=True):
-            local('pip install ansible==1.9.4')
-            local('sudo ansible-galaxy install -r $FAB_PATH/../orchestration/roles/roles.txt')
+            local('pip install ansible')
+        with settings(warn_only=True):
+            local('sudo ansible-galaxy remove nodesource.node')
```

Comment:
Could this have some unintended consequences on removing the role for another project that's using it? Basically could this break the deployment of another app that's maybe depending on an older version of node or this role?